src/hexset.rs

```diff
@@ -0,0 +1,81 @@
+//! A HexMap is a structure for representing geographical regions and
```

Comment:
Because this module is private you should move these docs into the the root module